.github/workflows/upload-source-maps.yml

```diff
@@ -1,3 +1,4 @@
+# TODO @tom2drum setup source maps for Rollbar
```

Comment:
This will be addressed in the following task, provided the tool functions as needed. Currently, the documentation does not clarify whether we can upload one source map for all instances. It states that we must specify a base URL to match the stack frame location.